src/applications/personalization/dashboard/tests/containers/DashboardApp.unit.spec.jsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
+import sinon from 'sinon';
 import { expect } from 'chai';
 
+import localStorage from '../../../../../platform/utilities/storage/localStorage';
 import { DashboardApp } from '../../containers/DashboardApp';
 
-
 const props = {
   profile: {
     loa: {}
   }
 };
 
 describe('<DashboardApp>', () => {
-  it('should render', () => {
-    window.localStorage = {
-      getItem: () => {}
-    };
+  before(() => {
```

Comment:
I'm actually unsure of why this wasn't causing an error previously, as the former value of localStorage isn't being restored.